docs/rustdoc/centralized_telescope/params/setup.md

```diff
@@ -0,0 +1,89 @@
+## Parameter generation protocol
+### Initialization
+1. Set the proof size $u$:
+
+$$
+u \coloneqq \left\lceil \frac{\lambda_{\text{sec}} + \log \lambda_{\text{rel}} + 5 - \log \log e}{\log \left(\frac{n_p}{n_f}\right)} \right\rceil
+$$
+
+2. Set the check values $s_1$ and $s_2$:
+
+$$
+ratio \coloneqq \frac{9 n_p \log e}{17u^2}
+$$
+
+$$
+s_1 \coloneqq ratio - 7, \quad s_2 \coloneqq ratio - 2.
+$$
+
+3. Set the values of $\lambda_{rel}^{(1)}$ and $\lambda_{rel}^{(2)}$:
+    - If $s_1 < 1 \implies \lambda_{rel}^{(1)} \coloneqq \bot$, else $\implies \lambda_{rel}^{(1)} \coloneqq \mathsf{min}(\lambda_{rel}, s_1)$
+    - If $s_2 < 1 \implies \lambda_{rel}^{(2)} \coloneqq \bot$, else $\implies \lambda_{rel}^{(2)} \coloneqq \mathsf{min}(\lambda_{rel}, s_2)$
+
+### Cases
```

Comment:
The cases here don't really correspond to small, intermediate and high cases that you outlined in `strategy.md`. The intermediate case there implies the number of repetitions r > 1, but cases 2 and 3 in this file can in principle set r = 1. So, cases 2 and 3 here really cover both intermediate and high cases but in different ways, depending on how u and lam_rel^(2) compare. Suggest changing wording.

Reply:
@curiecrypt Just precise here that we use a different comparison for the intermediary and high cases instead of lambda^2 <= u < lambda^3 and u >= lambda^3

Reply:
I think the documentation should provide a bridging content between the code and the theory. I tried to create an intermediate-level text for someone who looks at the code first. Matching structure with code is more convenient to understand the content of the paper, I guess.

Reply:
This is the biggest issue right now that prevent me from LGTMing.

Reply:
@rrtoledo